woocommerce/woocommerce-ios
# Store creation: update options in profiler questions and track event for skip actions

Closes: #9397 


## Description


With the goal to relaunch profiler questions in release 13.3, this PR includes implementation for a few tasks:
- Updated profiler industry options & order to match the Ghidorah design Y5pUYSJPsGEud1vknUZhi8-fi-191:16381
- Added new options for the selling platforms profiler question based on https://github.com/Automattic/woocommerce.com/pull/16687/files
- Tracked an event when the user skips a profiler question p1681807739886389/1681395877.201629-slack-C045CUK1Y3U - `site_creation_profiler_question_skipped` event with an event property `step`:
  - When skipping the category/industry question: `store_profiler_industries`
  - When skipping the selling status question: `store_profiler_commerce_journey`
  - When skipping the selling platforms after selecting "I'm already selling online" as the selling status: `store_profiler_ecommerce_platforms`

## Testing instructions


- Start the store creation flow either from the prologue or the store picker
- Enter a store name and continue --> the category profiler question should be shown with options that match the Ghidorah design
- Tap `Continue` or `Skip` without selecting an option --> `site_creation_profiler_question_skipped` event should be tracked with `step: store_profiler_industries`, and the selling status question should be shown
- Tap `Continue` or `Skip` without selecting an option --> `site_creation_profiler_question_skipped` event should be tracked with `step: store_profiler_commerce_journey`, and the store country question should be shown
- Navigate back to the selling status question
- Tap `I am already selling online` --> the selling platforms question should be shown with the new options
- Tap `Continue` or `Skip` without selecting an option --> `site_creation_profiler_question_skipped` event should be tracked with `step: store_profiler_ecommerce_platforms`, and the store country question should be shown

## Screenshots


updated categories | updated selling platforms (1/2) | updated selling platforms (2/2)
-- | -- | --
![Simulator Screen Shot - iPhone 14 - 2023-04-20 at 14 13 12](https://user-images.githubusercontent.com/1945542/233274316-f38f06ce-64b0-4cee-978b-67904bcdb4e7.png) | ![Simulator Screen Shot - iPhone 14 - 2023-04-20 at 14 13 33](https://user-images.githubusercontent.com/1945542/233274326-bea523b5-01dc-4fd7-9135-0e7437d7e425.png) | ![Simulator Screen Shot - iPhone 14 - 2023-04-20 at 14 13 36](https://user-images.githubusercontent.com/1945542/233274328-2eb814f2-683c-4b5a-8c5b-da84f9c498bc.png)


---
